### DISCUSSION TITLE
Email alerts not working? on wordpress plugin free version

### DISCUSSION THREAD

**Discussion by cctravis1** [quillforms/quillforms#114] [Q&A]
Is there a trick to getting email alerts for submissions working on the free version? We are considering the paid version but need to test things out first. Love the forms just need to figure out the emails. Hope the alerts are an easy fix as these are the best-looking forms we have found. 

The alert was set up and emails added but in the testing, nothing is showing up after form submission takes place.

**Comment by mdmag** [comment]
@cctravis1,

Sorry for being so late for this issue.
Please refer to this doc to troubleshoot email notifications: 
https://quillforms.com/docs/how-can-i-get-results-in-the-free-version/
